src/clients/AcrossConfigStoreClient/AcrossConfigStoreClient.ts

```diff
@@ -234,6 +235,31 @@ export class AcrossConfigStoreClient extends BaseAbstractClient {
     return this.isValidConfigStoreVersion(version);
   }
 
+  /**
+   * Return first block number where UBA was activated by setting "Version" GlobalConfig in ConfigStore
+   * @returns
+   */
+  getUbaActivationBlock(): number {
+    const config = this.cumulativeConfigStoreVersionUpdates.find((config) => {
+      isUBA(Number(config.value));
+    });
+    if (config) {
+      return config.blockNumber;
+    } else {
+      return Number.MAX_SAFE_INTEGER;
+    }
+  }
+
+  public isUbaBlock(block: number): boolean {
+    const versionAppliedToDeposit = this.getConfigStoreVersionForBlock(block);
+    return isUBA(versionAppliedToDeposit);
+  }
+
+  public canSupportUbaBlock(block: number): boolean {
+    const versionAppliedToDeposit = this.getConfigStoreVersionForBlock(block);
+    return this.isValidConfigStoreVersion(versionAppliedToDeposit);
+  }
+
```

Comment:
Thoughts on relocating these to `UBAClientUtilities.ts`?

Reply:
will do good call